Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -40,7 +45,13 @@ final class PackageCommandTests: CommandsTestCase {
         var environment = env ?? [:]
         // don't ignore local packages when caching
         environment["SWIFTPM_TESTS_PACKAGECACHE"] = "1"
```

Comment:
question: out of curiosity, what behavioural changes does setting this environment variable have?